src/app/static/src/tests/components/login.test.ts

```diff
@@ -0,0 +1,148 @@
+            "Veuillez entrer votre mot de passe.", "Por favor, introduza a sua palavra-passe.", store);
+        expectTranslatedWithStoreType<TranslatableState>(wrapper.find("#forgot-password>a"), "Forgotten your password?",
+            "Vous avez oublié votre mot de passe ?", "Esqueceu-se da sua palavra-passe?", store);
+
+        // Log In button
+        expectTranslatedWithStoreType<TranslatableState>(wrapper.find(".btn-red"), "Log In",
+            "Ouvrir une session", "Iniciar Sessão", store);
+
+        // No account
+        expectTranslatedWithStoreType<TranslatableState>(wrapper.find("#register-an-account>div"), "Don't have an account?",
+            "Vous n'avez pas de compte ?", "Não tem uma conta?", store);
+        expectTranslatedWithStoreType<TranslatableState>(wrapper.find("#requestAccount"), "Request an account",
+            "Demander un compte", "Solicite uma conta", store);
+
+        // Continue as guest
+        expectTranslatedWithStoreType<TranslatableState>(wrapper.find("#continue-as-guest>div"), "OR",
+            "OU", "OU", store);
+        expectTranslatedWithStoreType<TranslatableState>(wrapper.find("#continue-as-guest>a"), "Continue as guest",
+            "Continuer en tant qu'invité", "Continuar como convidado", store);
+
+        expect(wrapper.find("#error").exists()).toBe(false)
+
+        const links = wrapper.findAll("a")
+        expect(links.length).toBe(9)
+        expect(links.at(0).attributes("href")).toBe("https://www.unaids.org")
+        expect(links.at(0).find("img").attributes("src")).toBe("public/images/unaids_logo.png")
+
+        expect(links.at(1).attributes("href")).toBe("/password/forgot-password/")
+        expect(links.at(2).attributes("href")).toBe("https://forms.office.com/r/7S9EMigGr4")
+        expect(links.at(3).attributes("href")).toBe("/")
+
+        expect(links.at(4).attributes("href")).toBe("https://www.fjelltopp.org")
+        expect(links.at(4).find("img").attributes("src")).toBe("public/images/fjelltopp_logo.png")
+        expect(links.at(5).attributes("href")).toBe("https://www.imperial.ac.uk")
+        expect(links.at(5).find("img").attributes("src")).toBe("public/images/imperial_logo.png")
+        expect(links.at(6).attributes("href")).toBe("https://github.com/reside-ic")
+        expect(links.at(6).find("img").attributes("src")).toBe("public/images/reside_logo.png")
+        expect(links.at(7).attributes("href")).toBe("https://www.avenirhealth.org")
+        expect(links.at(7).find("img").attributes("src")).toBe("public/images/avenir_logo.png")
+        expect(links.at(8).attributes("href")).toBe("https://www.washington.edu")
+        expect(links.at(8).find("img").attributes("src")).toBe("public/images/uw_logo.png")
+    });
+
+    it("renders form with error", () => {
+        const store = createStore();
+
+        const wrapper = createSut(store, "Naomi", "/", "", "test error");
+        expect(wrapper.find("#error").text()).toBe("test error")
```

Comment:
For some reason, whenever I try to test the translated errors, only the translation key comes through (eg, "badUsernamePassword"). It works as expected when testing manually though. To test, enter a correct password and change the language.